xmodule/modulestore/xml.py

```diff
@@ -65,10 +65,8 @@ def __init__(self, xmlstore, course_id, course_dir,  # lint-amnesty, pylint: dis
         self.load_error_blocks = load_error_blocks
         self.modulestore = xmlstore
 
-        def process_xml(xml):  # lint-amnesty, pylint: disable=too-many-statements
-            """Takes an xml string, and returns a XBlock created from
-            that xml.
-            """
+        def process_xml(xml, source_info="unknown"):
+            """Takes an xml string, and returns a XBlock created from that xml."""
```

Comment:
The `source_info` doesn't seem to be used anywhere. This change should be reverted.